src/loaders/svg/tvgXmlParser.cpp

```diff
@@ -467,6 +467,12 @@ bool simpleXmlParseW3CAttribute(const char* buf, simpleXMLAttributeCb func, cons
     do {
         char* sep = (char*)strchr(buf, ':');
         next = (char*)strchr(buf, ';');
+        if (sep >= end)
+        {
+            next = nullptr;
+            sep = nullptr;
+        }
+        if (next >= end) next = nullptr;
```

Comment:
till now this func got data like:
`fill: red; stroke: green`
since I'd like to use the same func to read the css style data like 
`fill: red; stroke: green }
circle.red { fill: red; }
I have to explicitly check, whether I didn't go outside the scope of one style (circle.red defines a separate style)